src/lib/db.js

```diff
@@ -230,7 +230,7 @@ export class Database {
       if (user_or_org.length > 0 && !isReserved.check(user_or_org)) {
         let creator_url = `https://github.com/${user_or_org}`;
         await new Creator(creator_url, user_or_org).save();
-        await this.connectActivityToCreator(a.url, creator_url);
+        await this.connectLogToCreator(a.url, creator_url);
```

Comment:
This namechange makes no sense. It doesn't contain "Log" to creator, it gives an Activity object in the database a creator.

Reply:
Check new commits.